src/main/kotlin/no/nav/familie/ef/søknad/mapper/Oversettelser.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.søknad.mapper
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.kontrakter.felles.objectMapper
+import org.slf4j.LoggerFactory
+
+fun String.hentTekst(): String = Oversettelser.hentTekst(this)
+
+val kontekst = ThreadLocal<String>()
+
+object Oversettelser {
+
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    private val tekst: Map<String, Map<String, String>> =
+            objectMapper.readValue(javaClass.getResource("/tekst/tekst.json").readText())
+
+    fun hentTekst(key: String): String {
+        return tekst[hentSpråk()]?.get(key) ?: run {
```

Comment:
Hvis språk er en enum i stedet ser jeg kanskje inget behov for metoden hentSpråk()

Reply:
Sjekker :)